src/backend/InvenTree/build/serializers.py

```diff
@@ -205,7 +207,7 @@ class Meta:
     quantity = serializers.DecimalField(
         max_digits=15,
         decimal_places=5,
-        min_value=0,
+        min_value=Decimal(0),
```

Comment:
Q (non blocking): Is this essential to revisions?

Reply:
No, looks like these have bled through from my attempts to support the new django-rest-framework requirements. Not essential here, but doesn't hurt either